GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/DefinedItem.java

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2016 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.x3d;
+
+import java.util.concurrent.Future;
+
+import org.gearvrf.GVRMaterial;
+
+/*
+ * Used in Array List for items DEFined in X3D (using the DEF="...." parameter).
+ * Another item using the USE="...." parameter will search the array list for the 
+ * matching item.
+ */
+
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRRenderData;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRTexture;
+
+public class DefinedItem {
```

Comment:
Added javadoc comments.  As public, GearVRf's X3D implementation enables developers to access any DEFined object, similar to HTML5's DOM's (Document Object Model) getElementsByTagName("_def_name_here_"); method.  Currently, only GVRSceneObject allows access by name, but in X3D the lights, 3d meshes, texture shoul all be accessed by name.